### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Sample Cards]: The Luminosity Ratio of the focusable list items present in the dropdown under 'Sample Cards' is 1.3:1 which is less than the required 3:1.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8463]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

​Test Environment:​
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1
Tools: Logitech Bluetooth Keyboard
Tools: Color Contrast Analyzer

Repro Steps:

- Launch the Adaptive cards app on Android.
- Navigate to 'Import' control and select it.
- Navigate to 'Sample cards' and select it.
- Navigate to available dropdown under 'Sample cards'.
- Run the CCA tool.
- Observe the issue.

Actual Result:
The Luminosity Ratio of the focusable list items present in the dropdown under 'Sample Cards' is 1.3:1 which is less than the required 3:1.

Expected Result:
The Luminosity Ratio of the focusable list items present in the dropdown under 'Sample Cards' should be equal to or more than 3:1. 
A dark outline border should be sufficient for such an item.

User Experience:
Users with AT would be affected if the Luminosity Ratio of the list items present in the dropdown under 'Sample Cards' is less than the required 3:1.

WCAG Reference:
(https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.04.11)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-raians%40microsoft.com%7c6a2dded50c1c45e12a5608d9798bb984%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637674463301170115%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=3StFqNyebFAcprbeS8ca9BgKUtOHDRu8WJIr5iVU%2fXc%3D&reserved=0) “

### Screenshots

<img width="572" alt="MAS1 4 11-The Luminoscity Ratio of the list items present in the dropdown under 'Sample Cards' is 1 31 which is less than the required 31" src="https://user-images.githubusercontent.com/89897257/232703262-6f119603-ed03-478d-b9b1-92a3e3641546.png">

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
We don't fix issues on sample app as we don't ship it to the public.

**Comment by vagpt** [comment]
As per the above comment closing this issue.
